.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Task
+about: When a user story has to be broken into smaller engineering chunks, use a task!
+title: 'Task: Part of #Issue'
+labels: ''
+assignees: Sgtpluck
```

Comment:
[question] What are the odds that you end up getting accidentally assigned a bunch of tasks because you put your name in the template?  :)

Reply:
ugh i removed myself from it, i think there is a little bug in their creation system, i'll manually take myself out of this